docs/src/refs.bib

```diff
@@ -13,6 +167,105 @@ @Article{GHLMP96
   doi           = {10.1007/BF01190329}
 }
 
+@Article{GP92,
+  author        = {Geck, Meinolf and Pfeiffer, G{\"o}tz},
+  title         = {Unipotent characters of the {Chevalley} groups $D_4(q)$, $q$ odd},
+  zbl           = {0813.20016},
+  journal       = {Manuscr. Math.},
+  fjournal      = {Manuscripta Mathematica},
+  volume        = {76},
+  number        = {3-4},
+  pages         = {281--304},
+  year          = {1992},
+  doi           = {10.1007/BF02567762},
+  url           = {https://eudml.org/doc/155755},
+  language      = {English},
+  zbmath        = {219904}
+}
+
+@Article{Gec90,
+  author        = {Geck, Meinolf},
+  title         = {Irreducible {Brauer} characters of the 3-dimensional special unitary groups in non-defining
+                  characteristic},
+  zbl           = {0696.20011},
+  journal       = {Commun. Algebra},
+  fjournal      = {Communications in Algebra},
+  volume        = {18},
+  number        = {2},
+  pages         = {563--584},
+  year          = {1990},
+  doi           = {10.1080/00927879008823932},
+  language      = {English},
+  zbmath        = {4140018}
+}
+
+@Article{Gre55,
+  author        = {Green, James A.},
+  title         = {The characters of the finite general linear groups},
+  zbl           = {0068.25605},
+  journal       = {Trans. Am. Math. Soc.},
+  fjournal      = {Transactions of the American Mathematical Society},
+  volume        = {80},
+  pages         = {402--447},
+  year          = {1955},
+  doi           = {10.2307/1992997},
+  language      = {English},
+  zbmath        = {3115940}
+}
+
+@Article{HS77,
+  author        = {Hotta, R. and Springer, T. A.},
+  title         = {A specialization theorem for certain {Weyl} group representations and an application to the {Green}
+                  polynomials of unitary groups},
+  zbl           = {0389.20037},
+  journal       = {Invent. Math.},
+  fjournal      = {Inventiones Mathematicae},
+  volume        = {41},
+  pages         = {113--127},
+  year          = {1977},
+  doi           = {10.1007/BF01418371},
+  url           = {https://eudml.org/doc/142488},
+  language      = {English},
+  zbmath        = {3605124}
+}
+
+@Book{Hi90,
+  author        = {Hi{\ss}, Gerhard},
+  title         = {Zerlegungszahlen endlicher {Gruppen} vom {Lie}-{Typ} in nicht-definierender {Charakteristik}.},
+  zbl           = {0832.20021},
+  publisher     = {Aachen: Techn. Hochsch. Aachen, Mathematisch-Naturwiss. Fakult{\"a}t},
+  year          = {1990},
+  language      = {German},
+  zbmath        = {51240}
```

Comment:
Could at least link to https://publications.rwth-aachen.de/record/821447

And then we could have a `note` or so which says "See also ..." and points to https://link.springer.com/chapter/10.1007/978-3-0348-8658-1_17 (and we should add that to the bib, too -- at the bottom of the page there is citation info, or perhaps zbmath or mathscinet has it, too)

Reply:
> And then we could have a note or so which says "See also ..." and points to https://link.springer.com/chapter/10.1007/978-3-0348-8658-1_17 (and we should add that to the bib, too -- at the bottom of the page there is citation info, or perhaps zbmath or mathscinet has it, too)

I'm sorry, I don't get it. Where could we add a note about what? Something like:
```suggestion
  zbmath        = {51240},
  note = {See also: https://link.springer.com/chapter/10.1007/978-3-0348-8658-1_17}
```